src/chapters/azure-intro/studio-2.rst

```diff
@@ -0,0 +1,75 @@
+===================================================
+Deploying & Connecting to a Machine on the same LAN
+===================================================
+
+As an additional example, you will deploy your web app to your local machine that is connected to a LAN and then access the swagger front end using your phone, or another device you have connected to your LAN.
+
+To complete this studio you will need your development machine, and a separate device that has access to a web browser and both your local machine and the separate device must be conncected to the same LAN.
+
+This studio will be very similar to the previous studio, but instead of relying on an instructor you will be playing both parts yourself. You will need to go through the development, and remote server instructions again, and then finally you will need to connect to the deployed application by using another device on your network.
+
+The Development Machine
+-----------------------
+
+As the **development machine** you are simulating where the source code is developed (your local machine).
+
+Open the ``CodingEventsAPI/Startup.cs`` file. again edit the code in some way so when you connect from the other device you will see the changes.
+
+.. warning::
+
+   If you do not see the code on this line you are likely on the wrong branch. Make sure you are working on the ``1-sqlite`` branch before continuing.
+
+You will see a configuration for Swagger on ``line 25``:
+
+.. sourcecode:: csharp
+   :lineno-start: 25
+   :emphasis-lines: 31,32
+   :caption: CodingEventsAPI/CodingEventsAPI.csproj
+
+   services.AddSwaggerGen(
+      options => {
+         options.SwaggerDoc(
+            "v1",
+            new OpenApiInfo {
+               Version = "v1",
+               Title = "Coding Events API",
+               Description = "REST API for managing Coding Events"
+            }
+         );
+      }
+   );
+
+Change the ``Title`` of the UI to include your name. You can also customize the ``Description`` if you would like. Then commit and push your changes:
+
+.. sourcecode:: bash
+   :caption: run from the root (solution) directory
+
+   $ git add .
+   $ git commit -m "made it my own!"
+   $ git push
+
+The Remote Server Machine
+-------------------------
+
+Your development machine will also act as the remote server. After completing the steps in the previous section go ahead and deploy the application.
```

Comment:
expand on this. first explain what the goal is. then describe the dev machine behaving as the remote host **as a matter of perspective** since it is being accessed from another device